src/tools/tidy/Cargo.toml

```diff
@@ -11,6 +11,7 @@ miropt-test-tools = { path = "../miropt-test-tools" }
 lazy_static = "1"
 walkdir = "2"
 ignore = "0.4.18"
+owo-colors = { version = "3.5.0", features = ["supports-colors"] }
```

Comment:
How did we pick this library? Cargo (for example) is using termcolor; it seems like avoiding a different library for coloring might be better.

Reply:
i just picked a random one. but it probably makes sense to switch to termcolor, i'll do that

Reply:
How should we do the color support detection? Should I just use `supports-color` as now or reimplement some primitive detection myself?

Reply:
`!CiEnv::is_ci()`, maybe?

Reply:
Actually, rust already depends on `owo-colors` through miri->ui_test->color-eyre. But I changed tidy to use termcolor anyways.